### PR TITLE
Fix check access to object in view

### PR DESCRIPTION
**Pull request by axzx**
Should be check access to the element, not for whole collection.

I am targetting this branch, because this a bugfix.

## Changelog

```markdown
### Security
- Fixed view - check specific item collection, not to the whole collection.
```
## Subject

The error occurs in the view that displays the individual elements of the relationship many-to-many.

### CODE REVIEW

File: Resources/views/CRUD/show_orm_many_to_many.html.twig [hunk 1]
```diff
@@ -16,7 +16,7 @@ file that was distributed with this source code.
     {% set route_name = field_description.options.route.name %}
         {% for element in value %}
             <li>
-                {% if field_description.hasassociationadmin and field_description.associationadmin.hasRoute(route_name) and field_description.associationadmin.hasAccess(route_name, value)%}
+                {% if field_description.hasassociationadmin and field_description.associationadmin.hasRoute(route_name) and field_description.associationadmin.hasAccess(route_name, element)%}
```
Comment (greg0ire) [inline Resources/views/CRUD/show_orm_many_to_many.html.twig]: Can you linebreak this, while you're at it?

**Comment by greg0ire** [comment]
Nice job. Travis will probably fail, see https://github.com/sonata-project/SonataAdminBundle/pull/4288

**Comment by axzx** [comment]
@greg0ire 
I found the same bug in other files.

**Comment by greg0ire** [comment]
Restarting Travis, it should be ok now.

**Pull request merged**